docs/ossm/injection/README.md

```diff
@@ -0,0 +1,100 @@
+
+> **_NOTE:_** Upstream Istio offers also Manual injection using the ` istioctl kube-inject` which is not supported by OpenShift Service Mesh.
+
+After adding a correct label (see below) on a namespace, any new pods that are created in that namespace will automatically have a sidecar added to them.
+
+Automatic injection occurs at the pod-level. You won’t see any change to the deployment itself. Instead, you’ll want to check individual pods (via `oc describe`) to see the injected proxy.
+
+### Controlling the injection policy
+When a `default` control plane revision is installed or a default tag exists, it's possible to use following labels on a namespace or a pod:
+| Resource | Label | Enabled value | Disabled value |
+| --- | --- | --- | --- |
+| Namespace | `istio-injection` | `enabled` | `disabled` |
+| Pod | `sidecar.istio.io/inject` | `"true"` | `"false"` |
+
+If you are using different name for a control plane revisions, revision specific labels are instead used by a matching `istio.io/rev` label. For example, for a revision named `canary`:
+| Resource | Enabled Label | Disabled Label |
+| --- | --- | --- |
+| Namespace | `istio.io/rev=canary` | `istio-injection=disabled` |
+| Pod | `istio.io/rev=canary` | `sidecar.istio.io/inject="false"` |
+
+If the `istio-injection` label and the `istio.io/rev` label are both present on the same namespace, the istio-injection label will take precedence.
+
+The injector is configured with the following logic:
+
+1. If either label (`istio-injection` or `sidecar.istio.io/inject`) is disabled, the pod is not injected.
+2. If either label (`istio-injection` or `sidecar.istio.io/inject` or `istio.io/rev`) is enabled, the pod is injected.
+
+### Deploying an app
+Prerequisites:
+- The OpenShift Service Mesh operator has been installed
+- An Istio CNI resource has been created
+
+1. Prepare `default` `istio.yaml`:
+    ```yaml
+    kind: Istio
+    apiVersion: sailoperator.io/v1alpha1
+    metadata:
+      name: default
+    spec:
+      namespace: istio-system
+      updateStrategy:
+        type: InPlace
+      version: v1.23.0
+    ```
+1. Create `default` Istio CR in `istio-system` namespace:
+    ```bash
+    oc apply -f istio.yaml
+    ```
+1. Deploy `sleep` app:
+    ```bash
+    oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/sleep/sleep.yaml
+    ```
+1. Verify both deployment and pod have a single container:
+    ```bash
+    oc get deployment -o wide
+    NAME    READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES            SELECTOR
+    sleep   1/1     1            1           16s   sleep        curlimages/curl   app=sleep
+    oc get pod -l app=sleep
+    NAME                     READY   STATUS    RESTARTS   AGE
+    sleep-5577c64d7c-ntn9d   1/1     Running   0          16s
+    ```
+1. Label the `default` namespace with `istio-injection=enabled`:
```

Comment:
Like I mentioned above, I'm a little leery of telling users to use `istio-injection` until tag support lands in the operator. It only works this way when your `Istio.spec.updateStrategy.type = RevisionBased` _and_ your `Istio.metadata.name = default`. In other words, it will work for this example but maybe won't work generally if a user is trying to take this example and generalize it.